Documentation/docs/learn/python_quick_start.md

```diff
@@ -0,0 +1,495 @@
+# Python Quick Start
+
+(quick-start)=
+
+## Installation
+
+To install the ITK Python package:
+
+```
+$ pip install itk
+```
+
+## Usage
+
+### Basic example
+
+Here is a simple python script that reads an image, applies a median image filter (radius of 2 pixels), and writes the resulting image in a file.
+
+```py
```

Comment:
py : python ? According to https://github.com/jincheng9/markdown_supported_languages#heres-a-full-list-of-supported-languages